app/utils/decorator_utils.py

```diff
@@ -0,0 +1,33 @@
+from app.database.models.user import UserModel
+
+
+# definition of decorator to validate user
+def user_validation(user_function):
+
+    def validator(*args, **kwargs):
+
+        #  check if the given paramenters are in form of dict (to get user_id)
+        if kwargs:
+
+            # if parameters are given in form of dict
+            user = UserModel.find_by_id(kwargs['user_id'])
+
+        else:
+
+            # if parameters are given as args
+            user = UserModel.find_by_id(args[0])
+
+        # verify if user exists
+        if user:
+
+            # verify if user has verified his/her email
```

Comment:
remove

Reply:
@ramitsawhney27  please specify what you want me to remove/.

Reply:
The unneeded comments for each statement.